.github/workflows/build.yml

```
@@ -54,7 +54,7 @@ jobs:
             # into the PR branch anyway.
             #
             # https://github.com/python/core-workflow/issues/373
-            git diff --name-only origin/$GITHUB_BASE_REF.. | grep -qvE '(\.rst$|^Doc|^Misc|^\.pre-commit-config\.yaml$|\.ruff\.toml$)' && echo "run_tests=true" >> $GITHUB_OUTPUT || true
+            git diff --name-only origin/$GITHUB_BASE_REF.. | grep -qvE '(\.rst$|^Doc|^Misc|^\.pre-commit-config\.yaml$|\.ruff\.toml$|\.md$|\.ini$)' && echo "run_tests=true" >> $GITHUB_OUTPUT || true
```

Most of the `.ini` files are `mypy.ini` that we can skip.

But one is `Tools/freeze/extensions_win32.ini`, should that trigger a build?

---

I guess yes, it should trigger a Windows build at least.
What is the preferable way to exclude this .ini file? Modify the regexp?

---

Yeah, exclude `mypy\.ini$`, something like this?
```suggestion
            git diff --name-only origin/$GITHUB_BASE_REF.. | grep -qvE '(\.rst$|^Doc|^Misc|^\.pre-commit-config\.yaml$|\.ruff\.toml$|\.md$|mypy\.ini$)' && echo "run_tests=true" >> $GITHUB_OUTPUT || true
```